builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -0,0 +1,64 @@
+package ldap
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/testhelpers/ldap"
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test relies on an external ldap server with a suitable person object (cn=admin,dc=planetexpress,dc=com)
```

Comment:
This comment lead me to think this is something that I'd need to set up as someone running the test. I think it's handled by `ldap.PrepareTestContainer()`?